### ISSUE TITLE
Textext can't find most components of Texlive-distribution

### ISSUE THREAD

**Issue by nameiwillforget** [textext/textext#416]
**Description of the problem/ steps to reproduce**
Textext seems not to find key components of my Texlive-distribution. It does however know the paths to the `pdflatex`, `xetex` and `lualatex` executables.

**Steps to reproduce the behavior (if not obvious from description)**:
Install and open Inkscape and Textext, then try to compile anything.

**Message(s) in error dialog (if this comes up)**
If I use `pdflatex`:

<details>
<summary>dialog output (if available):

```
! LaTeX Error: File `article.cls' not found.



Type X to quit or <RETURN> to proceed,
```

If I use `lualatex`:

```
! Emergency stop.

<everyjob> \directlua {require("lualatexquotejobname.lua")}

                                                \typeout {\fmtname \space <\fmt
```
</summary>
</details>

<details>
<summary>output of stdout (if available):

If I use `pdflatex`
```
This is pdfTeX, Version 3.141592653-2.6-1.40.25 (TeX Live 2023/Arch Linux) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./tmp.tex
LaTeX2e <2022-11-01> patch level 1
L3 programming layer <2023-02-22>

! LaTeX Error: File `article.cls' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: cls)

Enter file name: 
! Emergency stop.
<read *> 
         
l.3 ^^M
       
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on tmp.log.
```

If I use lualatex:
```
This is LuaHBTeX, Version 1.17.0 (TeX Live 2023/Arch Linux) 
 restricted system commands enabled.
(./tmp.tex[\directlua]:1: module 'lualatexquotejobname.lua' not found:
	no field package.preload['lualatexquotejobname.lua']
	[kpse lua searcher] file not found: 'lualatexquotejobname.lua'
stack traceback:
	[C]: in function 'require'
	[\directlua]:1: in main chunk.
<everyjob> \directlua {require("lualatexquotejobname.lua")}
                                                \typeout {\fmtname \space <\fmt
l.1 
  
? 
! Emergency stop.
<everyjob> \directlua {require("lualatexquotejobname.lua")}
                                                \typeout {\fmtname \space <\fmt
l.1 
  
 275 words of node memory still in use:
   1 hlist, 1 dir, 39 glue_spec nodes
   avail lists: 2:12,3:3,4:1
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on tmp.log.
```
</summary>
</details>

<details>
<summary>output of stderr (if available):</summary>

```
<<< REPLACE THIS LINE WITH ACTUAL OUTPUT >>> 
```
</details>

**Logfile**
[.cache.json](https://github.com/textext/textext/files/13980731/default.cache.json)

**System information**

- TexText version: 1.10.0
- Inkscape version: Inkscape 1.3.2 (091e20ef0f, 2023-11-25, custom)
- Operating system: Arch Linux, freshly upgraded.

**Comment by jcwinkler** [comment]
Thanks for the report. To compile the code you entered in the TexText window TexText just performs a system call `pdflatex` (for example). Everything what goes wrong during LaTeX compilation is out of the scope of TexText, it is a pure problem of the LaTeX distro.

In your case the call to `/usr/bin/pdflatex` (defined in your settings) seems not to invoke a well setup LaTeX distro.

Try to compile a small LaTeX document, e.g. `test.tex` using the command `/usr/bin/pdflatex test.tex` and you should run into the same errors.

BTW: Shouldn't texlive binaries not be placed in `/usr/local/texlive` ? What's the output of `which pdflatex`?

**Comment by nameiwillforget** [comment]
I found the error: prior to installing TeXLive natively I installed it using the package manager Pacman and because Pacman requires its own installation as a dependency of some packages I wanted to keep being updated by it, I kept Pacman's distribution installed. Textext found Pacman's distribution first and tried to use it. I didn't think that could be it because that distribution should still be functional, but apparently it was. I've now replaced Pacman's distribution with a dummy package as described here:

https://tex.stackexchange.com/questions/696777/how-to-fake-packages-for-tex-live-on-arch-linux/696839#696839

Then I reinstalled Textext manually (I first tried to reinstall it using Pacman, but it still kept looking in the wrong path), and with Pacman's distribution gone it found the natively installed TeXLive distribution and works using it now. Maybe it would be a good idea to have Textext first look for native installations and only for others once it can't find one.
